doc/sphinx/addendum/type-classes.rst

```diff
@@ -322,12 +340,30 @@ Summary of the commands
    Like any command declaring a record, this command supports the
    :attr:`universes(polymorphic)`, :attr:`universes(template)`,
    :attr:`universes(cumulative)`, and :attr:`private(matching)`
-   attributes.
+   attributes. In addition, it supports the :attr:`mode` attribute.
+
+   .. attr:: mode
+
+      Sets the resolution mode of the class
+      at declaration time, using the same syntax as :cmd:`Hint Mode`,
```

Comment:
"Same syntax" means _everything_ after "Mode" below, or something less?  You can include the syntax inline in the paragraph with ```:n:`@qualid {+ {| + | ! | - } } {? : {+ @ident } }` ```

![image](https://user-images.githubusercontent.com/1253341/114759076-933aab80-9d12-11eb-9140-3eed8e62e7e3.png)

Reply:
Thanks!